server/task_executor_database_create.go

```diff
@@ -120,6 +120,57 @@ func (exec *DatabaseCreateTaskExecutor) RunOnce(ctx context.Context, server *Ser
 		return true, nil, err
 	}
 
+	// After the database creation statement executed rightly,
+	// insert its information to our database.
+	databaseCreate := &api.DatabaseCreate{
+		CreatorID:     api.SystemBotID,
+		ProjectID:     payload.ProjectID,
+		InstanceID:    task.InstanceID,
+		EnvironmentID: instance.EnvironmentID,
+		Name:          payload.DatabaseName,
+		CharacterSet:  payload.CharacterSet,
+		Collation:     payload.Collation,
+		Labels:        &payload.Labels,
+		SchemaVersion: payload.SchemaVersion,
+	}
+	database, err := server.DatabaseService.CreateDatabase(ctx, databaseCreate)
+	if err != nil {
+		return true, nil, err
+	}
+
+	// Update task database_id with the created database id
```

Comment:
This comment only says what, but it's more valuable to explain why.

Reply:
Updated, PTAL.